notebooks/quick_start/your_first_model.ipynb

```diff
@@ -0,0 +1,907 @@
+        "        loss = torch.nn.functional.cross_entropy(logits, label)\r\n",
+        "        return {'loss': loss, 'probs': probs}\r\n",
+        "</code></pre>\r\n",
+        "\r\n",
+        "After we have embedded our text, we next have to squash the sequence of vectors (one per token) into a single vector for the whole text. We do that using the `Seq2VecEncoder` that we got as a constructor argument. In order to behave properly when we’re batching pieces of text together that could have different lengths, we need to mask elements in the `embedded_text` tensor that are only there due to padding. We use a utility function to get a mask from the `TextField` output, then pass that mask into the encoder.\r\n",
+        "\r\n",
+        "At the end of these lines, we have a single vector for each instance in the batch."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "uUxYty0Y_7A-"
+      },
+      "source": [
+        "### Making predictions\r\n",
+        "\r\n",
+        "<pre class=\"brush: python\"><code class=\"language-python\">\r\n",
+        "class SimpleClassifier(Model):\r\n",
+        "    def forward(self,\r\n",
+        "                text: TextFieldTensors,\r\n",
+        "                label: torch.Tensor) -> Dict[str, torch.Tensor]:\r\n",
+        "        # Shape: (batch_size, num_tokens, embedding_dim)\r\n",
+        "        embedded_text = self.embedder(text)\r\n",
+        "        # Shape: (batch_size, num_tokens)\r\n",
+        "        mask = util.get_text_field_mask(text)\r\n",
+        "        # Shape: (batch_size, encoding_dim)\r\n",
+        "        encoded_text = self.encoder(embedded_text, mask)\r\n",
+        "        <strong># Shape: (batch_size, num_labels)\r\n",
+        "        logits = self.classifier(encoded_text)\r\n",
+        "        # Shape: (batch_size, num_labels)\r\n",
+        "        probs = torch.nn.functional.softmax(logits)\r\n",
+        "        # Shape: (1,)\r\n",
+        "        loss = torch.nn.functional.cross_entropy(logits, label)</strong>\r\n",
+        "        return {'loss': loss, 'probs': probs}\r\n",
+        "</code></pre>\r\n",
+        "\r\n",
+        "The last step of our model is to take the vector for each instance in the batch and predict a label for it. Our `classifier` is a `torch.nn.Linear` layer that gives a score (commonly called a `logit`) for each possible label. We normalize those scores using a `softmax` operation to get a probability distribution over labels that we can return to a consumer of this model. For computing the loss, PyTorch has a built in function that computes the cross entropy between the logits that we predict and the true label distribution, and we use that as our loss function.\r\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "DQVGPAu0A_iA"
+      },
+      "source": [
+        "# class SimpleClassifier(Model)"
```

Comment:
This looks a bit weird as the header.. maybe could just call this section "Putting it all together" or something like that.